pages/dkp/konvoy/2.2/release-notes/2.2.3/index.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: layout.pug
+navigationTitle: DKP 2.2.3 Release Notes
+title: DKP 2.2.3 Release Notes
+menuWeight: 30
+excerpt: View release-specific information for DKP 2.2.3
+enterprise: false
+beta: false
+---
+**D2iQ&reg; Konvoy&reg; version 2.2.3 was released on October 14th, 2022.**
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download DKP[/button]
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install DKP.</p>
+
+## Release summary
+
+Welcome to D2iQ Kubernetes Platform (DKP) 2.2.3! This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintains compatibility and support for other packages used in Konvoy.
+
+DKP 2.2.x supports Kubernetes versions between 1.21.0 and 1.22.x. Any cluster you want to attach using DKP 2.2.x must be running a Kubernetes version in this range.
+
+### Supported versions
+
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**  | 1.21.0  |
+| **Maximum**  | 1.22.x  |
+| **Default**  | 1.22.8  |
+
+## Fixes and Improvements
+
+### Chartmuseum not hosting with the cert provided to Kommander during installation, x509 unknown error on Workload cluster (D2IQ-93002)
+
+[Looking at the ticket, do we have a fix for this? We've resolved CA certs below but not the connected missing airgap bundle ticket.]
+
+### Cannot resolve CA certificate when provided as custom certificate (D2IQ-91798)
+
+To resolve this issue, use a custom image for the Kommander federation controllers that includes the patch. The easiest way to do that is by using the installer config. This works both when:
+
+* Installing a new cluster
+
+* Re-running `dkp install kommander` on an existing v2.2.2 cluster
+
+In the installer configuration, under apps, add the following lines:
+```
+  kommander:
+    enabled: true
+    values: |
+      controller:
+        containers:
+          manager:
+            image:
+              repository: mesosphere/kommander2-federation-controller-manager
+              tag: <patched_image_tag>
+```
+### No apparent upgrade path for 1.X vSphere clusters to 2.X (D2IQ-90769)
+
+vSphere clusters can now be migrated from 1.X to 2.X versions of DKP.
+
+## Component and Application updates
+
+When upgrading to this release, the following services and service components are upgraded to the listed version:
+
+### Components
+
+- CAPPPVersion = "v0.8.3"
+- CAPZVersion  = "v1.3.1"
+- Containerd 1.4.13
+- KIB v1.17.2
+
+### Applications
```

Comment:
This is Konvoy, so we need to replace these applications with Konvoy Components and versions.

Reply:
Removed section @cmastr